frontend/src/pages/exploreApplication/GetStartedPanel.tsx

```diff
@@ -21,9 +21,7 @@ import { useAppContext } from '../../app/AppContext';
 import './GetStartedPanel.scss';
 
 const DEFAULT_BETA_TEXT =
-  'This application is available for early access prior to official ' +
-  ' release. It won’t appear in the *Enabled* view, but you can access it by' +
-  ' [signing up for beta access.](https://www.starburst.io/platform/starburst-galaxy/).';
+  'This application is available for early access prior to official release.';
```

Comment:
Is docs and UX okay with this wording?

Reply:
Content suggestion: 
“**Test App is in beta.**
Test App is available for early access in a beta pre-release stage. You might find bugs or issues with availability, stability, data, or performance.”

I just restated that the app is in beta in the description, and added a sentence about the drawbacks of using the app in beta. I feel like that’s probably important so that users know that they may not want to use the app in production. WDYT?

Reply:
Looks great to me, thanks @kaedward!